README.md

```diff
@@ -54,7 +54,7 @@ Argument | Example
 gamephysics_diff | 2D Rigid Body Physics Library
 swhs_diff | Solar Water Heating System with PCM
 glassbr_diff | Glass-BR
-tiny_diff | HGHC toy example
+HGHC_diff | HGHC toy example
```

Comment:
It has to be match case (e.g., `hghc_diff` instead) to match the Makefile target I think.